src/Main.elm

```diff
@@ -295,6 +298,57 @@ update rawMsg ({ state } as model) =
                 ( VersionPoll, _ ) ->
                     ( model, Request.Version.loadVersion VersionReceived )
 
+                -- Login
+                ( LoggedIn (Ok newProcessesJson), _ ) ->
+                    let
+                        newSession =
+                            Session.loggedIn session newProcessesJson
+                                |> Session.notifyInfo "Vous avez maintenant accès au détail des impacts" ""
```

Comment:
For some reason, this text is rendered in a very large fashion in the info notification

![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/7c10e182-4669-468c-8c51-048d97c9a43a)

It uses a `h5` while it should use a `h6`, and needs removing the bottom margin by adding the `mb-0` class to the element.

Also if you log in and out multiple times, the same notification stacks

![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/5ab4b2cf-c63a-4fd3-87e8-bab754fa3592)

To solve this (and if we want to, I reckon we do) we could either make them disappeear after a short while or avoid adding another instance of an already present notification (eg. make the notification list a Set)

Note: this shouldn't block landing this PR. But please file a card so we don't forget to fix it.

Reply:
https://www.notion.so/fd0068a7db8348c584cd53a3eea399f2?v=af2332265058414da4bd3aa4d381ed5f&p=27110f9f18914855afd272ee51bfc1dd&pm=s

Reply:
@n1k0 did you by any chance already fix that?

Reply:
@magopian nope I don't think so

Reply:
I don't see any h5 or h6 anymore on `staging`, and i can't reproduce the multiple notifications, maybe the changes you did to "make the UI nicer" in the `auth` pages?